src/module/data/pseudo-documents/power-roll-effects/applied-effect.mjs

```diff
@@ -80,25 +88,124 @@ export default class AppliedPowerRollEffect extends BasePowerRollEffect {
           placeholder: n > 1 ? this.applied[`tier${n - 1}`].display : "",
           name: `${path}.display`,
         },
-        always: {
-          field: this.schema.getField(`${path}.always`),
-          value: this.applied[`tier${n}`].always,
-          src: this._source.applied[`tier${n}`].always,
-          name: `${path}.always`,
+        effects: {
+          field: this.schema.getField(`${path}.effects`),
+          value: this.applied[`tier${n}`].effects,
+          src: this._source.applied[`tier${n}`].effects,
+          widget: this.#effectWidget.bind(this),
+          name: `${path}.effects`,
         },
-        success: {
-          field: this.schema.getField(`${path}.success`),
-          value: this.applied[`tier${n}`].success,
-          src: this._source.applied[`tier${n}`].success,
-          name: `${path}.success`,
+      });
+    }
+  }
+
+  /* -------------------------------------------------- */
+
+  /**
+   *
+   * @param {DataField} field
+   * @param {FormGroupConfig} groupConfig
+   * @param {FormInputConfig<AppliedEffectSchema["effects"]>} inputConfig
+   * @returns {HTMLFieldSetElement}
+   */
+  #effectWidget(field, groupConfig, inputConfig) {
```

Comment:
Missing comment in annotation. Wasn't really clear to me what this method's purpose was. But also this is a lot of code for what is as far as I understand a select with a change listener. What is the benefit?